ember-welcome-page/src/template-registry.ts

```diff
@@ -0,0 +1,6 @@
+import type WelcomePageComponent from './components/welcome-page';
+
+export default interface EmberWelcomePageRegistry {
+  WelcomePage: typeof WelcomePageComponent;
+  'welcome-page': typeof WelcomePageComponent;
```

Comment:
The `welcome-page` is needed if we want to allow developers to invoke the component with either of these syntaxes:

```hbs
{{! Dynamic invocation }}
{{component "welcome-page"}}

{{! Curly-brace syntax }}
{{welcome-page}}
```

If we want to allow only the angle bracket syntax `<WelcomePage>`, then we can delete this line.